part8-clientSide/index.html

```diff
@@ -0,0 +1,77 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Part 8</title>
+  </head>
+  <body>
+    <div>
+      <h1>Task 1 - <small>redirect to page</small></h1>
+      <button onclick="redirectToPage()">Click</button>
+    </div>
+
+    <div>
+      <h1>Task 2 - <small>go back</small></h1>
+      <button onclick="goBack()">Go back</button>
+    </div>
+
+    <div>
+      <h1>Task 3 - <small>browser name and version</small></h1>
+      <h4 id="browserName"></h4>
+      <h4 id="browserVersion"></h4>
+    </div>
+
+    <div>
+      <h1>Task 4 - <small>screen width and height</small></h1>
+      <h4 id="screenWidth"></h4>
+      <h4 id="screenHeight"></h4>
+    </div>
+
+    <div>
+      <h1>Task 5 - <small>change bg color to yellow</small></h1>
+      <button onclick="changeBackgroundColor()">Click</button>
+      <button onclick="changeBackgroundColorBack()">...And back</button>
+    </div>
+
+    <div>
+      <h1>Task 6 - <small>set cookie name and value</small></h1>
+      <button onclick="setCookie()">Click</button>
+    </div>
+
+    <div>
+      <h1>Task 7 - <small>highlight paragraphs</small></h1>
+      <button onclick="highlightParagraphs()">Hightlight paragraphs</button>
+      <p>Paragraph</p>
+      <p>Paragraph</p>
+      <section>Not a Paragraph</section>
+      <p>Paragraph</p>
+      <section>Not a Paragraph</section>
+    </div>
+
+    <div>
+      <h1>Task 8 - <small>Replace paragraphs text</small></h1>
+      <button onclick="replaceParagraphText()">Replace paragraphs text</button>
+      <p id="replaceText">Paragraph</p>
+      <p id="replaceText">Paragraph</p>
+      <section id="replaceText">Not a Paragraph</section>
+      <p id="replaceText">Paragraph</p>
+      <section id="replaceText">Not a Paragraph</section>
+    </div>
+
+    <div>
+      <h1>Task 9 - <small>customize element</small></h1>
+      <button onclick="customizeElement()">Customize Element</button>
+      <div id="myElement">ELEMENT</div>
+    </div>
+
+    <div>
+      <h1>Task 10 - <small>trigger two different alerts</small></h1>
+      <button id="myButton">Click</button>
```

Comment:
in order to see event propagation you should move button inside the parent element, like so:

```html
<div id="parentElement">
    <button id="myButton">Click</button>
    <p>Parent Element</p>
</div>
```
Then try to click button. You will see in which order alerts are triggered.